oc4ids_datastore_pipeline/registry.py

```diff
@@ -16,13 +16,17 @@ def fetch_registered_datasets() -> dict[str, dict[str, str]]:
         r = requests.get(url)
         r.raise_for_status()
         json_data = r.json()
-        registered_datasets = {
-            key: {
-                "source_url": value["fields"]["url"]["value"],
-                "country": value["fields"]["country"]["value"],
+        registered_datasets = {}
+        for key, value in json_data["records"].items():
+            r_data = requests.get(value["api_url"])
```

Comment:
I'm not sure how big `json_data["records"]` is but that could be a very large number of requests all at once? Has needing some kind of control on that been considered?